core/src/commonIntegrationTest/kotlin/com/powersync/DatabaseTest.kt

```diff
@@ -97,7 +213,7 @@ class DatabaseTest {
                 try {
                     database.writeTransaction {
                         it.execute("DELETE FROM users;")
-                        it.execute("syntax error, revert please")
+                        it.execute("syntax error, revert please (this is intentional from the unit test)")
```

Comment:
This change sounds like it took some time to figure out where that error came from at some point. Sorry for the original text 🙈

Reply:
😄 I just added this for some clarity, it didn't take too long to see the log was coming from this test.